contributing/documentation/translations.rst

```diff
@@ -1,7 +1,14 @@
 Translations
 ============
 
-The official Symfony documentation is published only in English, but some
-community groups maintain `unofficial translations`_.
+The official Symfony documentation is published only in English. You can
+read about the reasons in `this blog post`_.
 
-.. _`unofficial translations`: https://symfony.com/blog/discontinuing-the-symfony-community-translations
+We have taken steps to improve the experience when using
+`google translate`_ to prevent code blocks from being translated.
```

Comment:
`google translate` -> `Google Translate`  ... and there's other occurrence below.

Reply:
fixed